walk-in-interview/src/main/webapp/new-job.js

```diff
@@ -0,0 +1,74 @@
+
+const NEW_JOB_FORM_ID = 'new-job-form';
+const NEW_JOB_TITLE_ID = 'new-job-title';
+const NEW_JOB_ADDRESS_ID = 'new-job-address';
+const NEW_JOB_DESCRIPTION_ID = 'new-job-description';
+const NEW_JOB_PAY = {
+  FREQUENCY_ID: 'new-job-pay-frequency',
+  MIN_ID: 'new-job-pay-min',
+  MAX_ID: 'new-job-pay-max',
+};
+const REQUIREMENTS_LIST_ID = 'requirements-list';
+const NEW_JOB_EXPIRY_ID = 'new-job-expiry';
+const NEW_JOB_DURATION_ID = 'new-job-duration';
+
+/**
+ * Adds a new job listing by making a POST request to the /jobs servlet.
+ * @return {boolean} If the request given the parameters was valid or not.
+ */
+function addJob() {
+  event.preventDefault();
```

Comment:
event is not defined in the arguments, `event` should be `undefined` here IIUC

this would work if your method signature has event argument
```
function addJon(event) {
    event.preventDefault();
}
```

And `Form#onSubmit` sends the event object.

Reply:
It seems that the event object is defined even without passing it in the arguments. I tried passing it as an argument but then when I did event.preventDefault(), it didn't work. Do you know why this might be?